src/igvfd/schemas/biosample.json

```diff
@@ -56,39 +55,6 @@
                 "lower_bound_age",
                 "age_units"
             ]
-        },
-        "age_units": {
-            "comment": "Human biosamples cannot have age units of minutes or hours. Yeast biosamples cannot have age units of years or months.",
-            "required": [
-                "lower_bound_age",
-                "upper_bound_age"
```

Comment:
Why was this removed? I think "lower_bound_age" and "upper_bound_age" should still be required for submitting "age_units."

Reply:
you're right, i re-added them